ui/lib/pki/addon/controllers/issuers/index.js

```diff
@@ -1,8 +1,12 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { next } from '@ember/runloop';
+import { getOwner } from '@ember/application';
 
 export default class PkiRolesIssuerController extends Controller {
+  get mountPoint() {
```

Comment:
in ember engines, we use mountPoints in the LinkedBlock component. It's used for the @linkPrefix param.